jmeter/upload-jmeter-scripts.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Usage: upload-jmeter-scripts.sh <hostname>
+# <hostname> can be an entry from your ~/.ssh/config
+# or it can be of form user:pwd@host (same format as in ssh).
+
+host=$1
+
+if [ -z $1 ]
+then
+  >&2 echo 'Usage: upload-jmeter-scripts.sh <hostname>'
+  exit 1
+fi
+
+# Create fresh directories, and copy jmeter files.
+ssh $1 'rm -rf ~/jmeter'
+
+ssh $1 'mkdir -p ~/jmeter'
+scp *.jmx $host:~/jmeter
+scp install-jmeter.sh $host:~/jmeter
+scp jmeter-version.sh $host:~/jmeter
+scp run-tests.sh $host:~/jmeter
+
+ssh $1 'mkdir -p ~/jmeter/fixtures'
+scp fixtures/* $host:~/jmeter/fixtures
```

Comment:
Instead of all of these individual scp commands, can we just run a recursive command to copy all of the files here (perhaps we need to exclude certain files). I think it would be helpful to have all of the source files including the README on the remote machine.

Reply:
Yeah we can upload all *.sh files at once.

Reply:
Actually, no, `upload-jmeter-scripts.sh` is not needed remotely, and neither is `run-gui.sh`. So at this time, there is no syntax gains for writing a pattern that excludes certain files.